js/main.js

```diff
@@ -0,0 +1,243 @@
+//loader functions
+function loaderDisplay() {
+    let loader = document.getElementById('loader');
+    loader.classList.remove('display-none')
+}
+
+function loaderRemove() {
+    let loader = document.getElementById('loader');
+    loader.classList.add('display-none')
+}
+
+let searchingBoxValue = document.querySelector('#box-digit');
+
+// const url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=AA&limit=10&exchange=NASDAQ`;
+
+const container = document.createElement('div');
+const searchClickBtn = document.querySelector('#search-button');
+let stockCompanyData = [];
+
+
+//companies data fetching and rendering
+async function stocksList() {
+    console.log(searchingBoxValue.value);
+    const url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${searchingBoxValue.value}&limit=50&exchange=NASDAQ`;
+    console.log(url);
+
+    loaderDisplay();
+
+    fetch(url)
+        .then(resp => {
+            loaderDisplay();
+            // console.log(resp);
+            return resp.json();
+        })
+        .then(
+
+            async (stocks) => {
```

Comment:
unnesessary async keword. if you are using the .then() syntax, you should not mix it with async await.